esphome/components/samsung/samsung.cpp

```diff
@@ -37,7 +32,7 @@ bool SamsungClimate::on_receive(remote_base::RemoteReceiveData data) {
     return false;
   }
 
-  ESP_LOGD(TAG, "Received Samsung A/C message");
+  ESP_LOGD(TAG, "Received Samsung A/C message size %" PRIu8, data.size());
```

Comment:
```suggestion
  ESP_LOGD(TAG, "Received Samsung A/C message size %" PRId32, data.size());
```
https://github.com/esphome/esphome/blob/dbf4c2c4dae43ee929471a26823db88f373576d6/esphome/components/remote_base/remote_base.h#L56